packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission.jsx

```diff
@@ -42,17 +42,17 @@ const POISubmission = observer(
         const { account_settings, getChangeableFields } = client;
         const { refreshNotifications } = notifications;
 
-        const handleSelectionNext = () => {
+        const handleSelectionNext = (should_show_manual = false) => {
```

Comment:
if this flags is to forward the client to manual, you don't need to check any of the logic below and we can 
```
if(should_show_manual) {
  setSubmissionService(service.code.manual);
  setSubmissionStatus(submission_status_code.submitting);
}

Reply:
now that we're using `POIContext` you don't need this change and we don't need to pass this function to `IdvFailed` or `IdvDocumentSubmit` 
You can access the context directly

Reply:
1) yes, we can go with such implementation, but it just increases the quantity of lines. so let’s stay with current implementation.
2) let’s leave it for refactoring cards 👍

Reply:
@yauheni-deriv i think @shahzaib-deriv has a point here. We can use the POIContext and dont need to pass it as prop in Idvfailed and IdvDocumentSubmit components. Can you check this

Reply:
Alright, finally I understood what you mean.
we can do this only for IdvDocumentSubmit, but not for IdvFailed, as IdvFailed is also used for mt5 flow.
so let’s not do it for both components for now just for better consistency 
@amina-deriv , @shahzaib-deriv